### ISSUE TITLE
[Feature] Allow user to load vATIS files from URL to support auto-update

### ISSUE THREAD

**Issue by fcooper94** [vatis-project/vatis#176]
I constantly see members in our division running ATIS files made for vATIS v1 and they never update their files, even when new updates are available. It would be great if you could load a vATIS file from a Github URL for example allowing the division to update the ATIS file remotely. 

With a notification when the file has been updated.

**Comment by justinshannon** [comment]
I’m currently working on revamping vATIS. I have disclosed much about it, but I am considering some sort of “data admin” site that facility staff can use to create and modify ATIS profiles. This will allow vATIS users to easily download the ATIS profiles and they will automatically update with any changes.

**Comment by ml0130** [comment]
Are you looking at `data admin` like what vNAS has, where the day-to-day controller can't edit or make changes to their atis files?

**Comment by ml0130** [comment]
> I’m currently working on revamping vATIS. I have disclosed much about it, but I am considering some sort of “data admin” site that facility staff can use to create and modify ATIS profiles. This will allow vATIS users to easily download the ATIS profiles and they will automatically update with any changes.


Are you looking at data admin like what vNAS has, where the day-to-day controller can't edit or make changes to their atis files?

**Comment by justinshannon** [comment]
> > I’m currently working on revamping vATIS. I have disclosed much about it, but I am considering some sort of “data admin” site that facility staff can use to create and modify ATIS profiles. This will allow vATIS users to easily download the ATIS profiles and they will automatically update with any changes.
> 
> Are you looking at data admin like what vNAS has, where the day-to-day controller can't edit or make changes to their atis files?

No, not necessarily. Maybe more of just a central repository? It would just be nice to have some sort of functionality to allow controllers to easily download and update their ATIS profiles without having to manually download them from their facility's website, similar to how CRC works in that regard.

If you have any suggestions, please feel free to share!

**Comment by ml0130** [comment]
Until then, the "data admin/repository" takes more shape. At this point, the only thing that comes to mind is if the user already has a profile set up for a facility, they can locally upload it and choose what it keeps up to date within the file. For example, in most of the facilities I run, I have added additional NOTAMs and Conjugations, and those would be details that I wouldn't want to be overwritten when the facility decides to update.

**Comment by fcooper94** [comment]
> I’m currently working on revamping vATIS. I have disclosed much about it, but I am considering some sort of “data admin” site that facility staff can use to create and modify ATIS profiles. This will allow vATIS users to easily download the ATIS profiles and they will automatically update with any changes.

Excellent - Having a central repo is what we need.
